nuxeo-coldstorage-package/src/main/resources/install/templates/coldstorage/nuxeo.defaults

```diff
@@ -1,5 +1,17 @@
 ## DO NOT EDIT THIS FILE, USE nuxeo.conf  ##
+nuxeo.template.includes=s3binaries
 #GLACIER
-s3-coldstorage.target=.
-nuxeo.glacier.bucket=
-nuxeo.glacier.bucket_prefix=
+coldstorage.target=.
+
+# mandatory
+nuxeo.aws.glacier.bucket=
+nuxeo.core.binarymanager=org.nuxeo.ecm.blob.s3.S3BlobProvider
+
+# optional
+nuxeo.aws.glacier.bucket_prefix=
+nuxeo.s3storage.cachesize=100MB
+nuxeo.s3storage.connection.max=50
+nuxeo.s3storage.connection.retry=3
+nuxeo.s3storage.connection.timeout=50000
+nuxeo.s3storage.socket.timeout=50000
+nuxeo.core.blobstore.digestAsync=false
```

Comment:
can you please move the `nuxeo.core.blobstore.digestAsync=false` near `nuxeo.core.binarymanager=org.nuxeo.ecm.blob.s3.S3BlobProvider` :)